script/scriptgen.py

```diff
@@ -701,21 +709,6 @@ def gen_read_files(self, f):
                             os.path.basename(repodir.attrib["folder"]).lstrip("*") + repodir.get("archs", "")
                         ),
                     )
-            elif "Factory" or "LEO" in self.ag.envdir:
-                for repodir in self.repodirs:
-                    if not repodir.attrib.get("gen", ""):
-                        self.p(
-                            cfg.read_files_repo_media + cfg.read_files_repo_media_convert,
-                            f,
-                            "PRODUCTREPOPATH",
-                            self.ag.productpath + "/" + self.folder + "/*" + repodir.attrib["folder"] + wild,
-                            "Media1.lst",
-                            "",
-                            "destlst",
-                            "Media1_{}.lst".format(os.path.basename(repodir.attrib["folder"]).lstrip("*")),
-                            "media.1/media",
-                            "media.1/products",
-                        )
```

Comment:
Won't this break projects that haven't switched yet?

Reply:
No it is only health check for Factory and LEO when repo path didn't have snapshot in it